src/app/registration-form/registration-form.component.ts

```diff
@@ -1,15 +1,12 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
strings in angular should be in '' not in "" (probably wrong prettier/formatter config?